README.md

```diff
@@ -1,3 +1,241 @@
-# watson-conversation-slots-intro
-
 ## Work In Progress. Stay tuned for forthcoming complete code.
+
+# Creating a Retail Chatbot using Watson Conversation, Discovery and Database Services
```

Comment:
Retail? I think we want to avoid that term in case its confused with our popular "Retail Chatbot" / Watson-Online-Store